src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -230,6 +230,9 @@ public void updateBranch() throws GitBranchException {
         }
     }
 
+    /**
+     * Gets the path to the root folder of the repository
```

Comment:
minor nits, just need to add full stop at the end of the sentence to keep it standardized.